### ISSUE TITLE
"freec -conf <config file> -sample <mySample.bam> -control <myControl.bam> " is not used

### ISSUE THREAD

**Issue by angeladarling** [BoevaLab/FREEC#105]
Hello:
I have tried use "freec -conf <config file> -sample <mySample.bam> -control <myControl.bam> " to input  Tbam and Nbam, but ,if find bam but not find Nbam ,what's the problem? Look forward to your reply ,thanks.

![image](https://user-images.githubusercontent.com/26453683/174247973-cadea67c-9f96-4b0f-8904-58fcaefaa44c.png)

![Uploading image.png…]()

**Comment by valeu** [comment]
Indeed, this is quite strange.. Can you share what you have in the config file?

**Comment by angeladarling** [comment]
> Indeed, this is quite strange.. Can you share what you have in the config file?

Please check ,thanks.

![image](https://user-images.githubusercontent.com/26453683/174509863-f5535eff-ff37-4abc-b0b6-cefbde133ae2.png)
